Solar/grobid_extraction/pdf_section_extraction/single_pdf/pdf_xml_extraction.py

```diff
@@ -42,20 +42,11 @@ def process_fulltext_document(service, file):
         os.makedirs(output_folder_xml)
 
     # Save XML file
-    output_path_xml = os.path.join(output_folder_xml, "paper_1.2.xml")
+    output_path_xml = os.path.join(output_folder_xml, "paper_15721500205e95c1bb016594.80962303.xml")
```

Comment:
I don't like hardcoded paths. Isn't there a CLI to use this?

Reply:
I don't know but that part just extract section for one paper, i can get rid of the hardcoded paths